src/components/Checkbox/CheckboxGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Checkbox from "@leafygreen-ui/checkbox";
+import { TreeDataEntry } from "components/TreeSelect";
+
+interface CheckboxesProps {
+  data: TreeDataEntry[];
+  value: string[];
+  onChange: (e: React.ChangeEvent<HTMLInputElement>, key: string) => void;
+}
+
+export const CheckboxGroup: React.FC<CheckboxesProps> = ({
```

Comment:
CheckboxGroup doesn't handle nested checkboxes that are found in the `children` of `TreeDataEntry`. Can you check with design if this is what we want to do moving forward?

Reply:
that feature can be added. my designs don't need it

Reply:
additionally, i would argue that the component for nested checkboxes in the form a tree should be its entirely own component since it is much more complex than this one.

Reply:
> that feature can be added in a future PR. my designs don't need it

We should add in the nested checkboxes as part of this ticket: https://jira.mongodb.org/browse/EVG-12550

> additionally, i would argue that the component for nested checkboxes in the form a tree should be its entirely own component since it is much more complex than this one. grouping everything into a single component for the sake of reusability can make code significantly less readable.

That makes sense to me. We should remove the word "tree" from variable names used throughout this PR since the ChecboxGroup doesn't support nested checkboxes.  https://jira.mongodb.org/browse/EVG-12550 can introduce a TreeCheckboxGroup.